tensorflow_datasets/core/utils/gpath.py

```diff
@@ -18,6 +18,7 @@
 import os
 import pathlib
 import posixpath
+from fnmatch import fnmatch
```

Comment:
Direct imports are not allowed (use `import fnmatch`, then `fnmatch .fnmatch()`)

Reply:
done